options/locale/locale_en-US.ini

```
@@ -1648,6 +1648,7 @@ pulls.push_rejected = Merge Failed: The push was rejected. Review the Git Hooks
 pulls.push_rejected_summary = Full Rejection Message
 pulls.push_rejected_no_message = Merge Failed: The push was rejected but there was no remote message.<br>Review the Git Hooks for this repository
 pulls.open_unmerged_pull_exists = `You cannot perform a reopen operation because there is a pending pull request (#%d) with identical properties.`
+pulls.open_unmerged_pull_merged = `You cannot perform a reopen operation because all the changes of this pull request have been merged into the base branch by another pull request.`
```

```suggestion
pulls.open_unmerged_pull_merged = `You cannot perform a reopen operation because the commits of this pull request have been merged into the base branch by another pull request.`
```

I assume this means exact commits, not diff, right?